app/views/comments/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<p style="color: green"><%= notice %></p>
+
```

Comment:
nitpick: you can delete empty lines in your view templates. Start your code at line 1. However, this does not affect the functionality of your app.